icu4c/source/data/dtd/cldr-40/common/dtd/ldmlICU.dtd

```diff
@@ -0,0 +1,119 @@
+<!--
```

Comment:
Why did you add these dtd files?

Reply:
See above comment

Reply:
There is "cldr-40" in the path. We already have these in a "cldr-35.1" path. Maybe we need these each time there is a certain change to the DTD?? Should the new ones *replace* the old ones?

Anyway, files like icu4c/source/data/xml/collation/root.xml refer to the new DTDs now. I didn't check if there are still files referring to the old ones.

Reply:
@pedberg could you please take a look at this question, too? This might be the one other place where we might want changes. (And why are these DTD files in a versioned path in the ICU repo anyway?)

Reply:
I do not know why they are in a versioned path. But there were dtd updates for 40 that were needed for the data/xml files.

Reply:
Any chnage we want here I would like to do in the next PR for alpha1 so I can merged this one before I start alpha1 integration.